### ISSUE TITLE
RN cycle dependencies warning

### ISSUE THREAD

**Issue by todorone** [LegendApp/legend-state#59]
In `react-native` project, that uses `pnpm`(not sure whether it's pnpm-specific), i've got such a warning:

```
 WARN  Require cycle: ../../node_modules/.pnpm/@legendapp+state@0.19.8/node_modules/@legendapp/state/react.js ->
../../node_modules/.pnpm/@legendapp+state@0.19.8/node_modules/@legendapp/state/react.js
```

This is not critical, but it would be nice to remove this for more comfortable RN use.

**Comment by jmeistrich** [comment]
Does it say `react.js` is requiring `react.js`? That seems odd... Can you try updating to `^0.21.8` and see if it's fixed in the latest version?

**Comment by todorone** [comment]
@jmeistrich Yeah, that's copy-pasted message from RN console, and 0.21.8 still throws. If the issue can't be reproduced on clean project with yarn/npm, than the issue is probably somewhere in-between `pnpm`/`Metro`. Feel free to ignore then, it may be loosely related to legend state.

**Comment by jmeistrich** [comment]
I think this should be fixed in `0.21.9`. There were some files within the `/react` export that were importing from the `/react` export (because they had been moved from a different export and I forgot to change that), so the built file was importing from itself.

**Comment by todorone** [comment]
@jmeistrich The issue was resolved, thanks.
